daal4py/sklearn/neighbors/tests/test_kneighbors.py

```diff
@@ -0,0 +1,106 @@
+#

```

Comment:
I would recommend not merge this now. Tests are also being prepared here https://github.com/IntelPython/daal4py/pull/280. it would be good to align them

Reply:
Tests are removed for now